### ISSUE TITLE
not detecting

### ISSUE THREAD

**Issue by Guacamole42** [SayakMukhopadhyay/EDMC-Discord-Presence#14]
I've installed the plugin to edmc, and I've also restarted edmc but it doesn't seem to detect it. the only way i can get it to even acknowledge that it's in the plugins folder is if i disable it

https://imgur.com/a/vAki14t

**Comment by kyanha** [comment]
EDMC 3.4.3.0, same issue.

**Comment by Poneyy** [comment]
it works fine in 3.5, but seems to have a missing module in 3.4.3

```
Traceback (most recent call last):
  File "plug.pyo", line 193, in load_plugins
  File "plug.pyo", line 94, in __init__
  File "C:\Users\Poneyy\AppData\Local\EDMarketConnector\plugins\DiscordPresence\load.py", line 22, in <module>
ImportError: No module named tkinter
```
Corrected by replace `Import tkinter` by `Import Tkinter` and all `.encode()` by `.encode('utf-8')`.
But it doesn't work in 3.5 with this.
```
  File "C:\Users\Poneyy\AppData\Local\EDMarketConnector\plugins\DiscordPresence\load.py", line 22, in <module>
    import Tkinter as tk
ModuleNotFoundError: No module named 'Tkinter'
```

**Comment by kyanha** [comment]
EDMC 3.5.0.0 breaks two other plugins I use.  I'm applying @Poneyy 's change to get it to work in 3.4.3.0 (which is also the current non-beta release version).

**Comment by RandomGgames** [comment]
same issue, 3.4.3.0. Seems like replacing tkinter with Tkinter in load.py gets it at least recognized by EDMC, and seems to be working fine. Replaced the .encodes() as well just in case. Nice find Poneyy :)

**Comment by SayakMukhopadhyay** [comment]
Sorry for the late response. But the latest version of the plugin supports only EDMC 3.5. Essentially the underlying Python was changed from 2 to 3. Use [Betelgeuse](https://github.com/SayakMukhopadhyay/EDMC-Discord-Presence/releases/tag/v1.1.0) if you want pre 3.5 support.

**Comment by SayakMukhopadhyay** [comment]
Closed by #15 
Use the [Gomeisa](https://github.com/SayakMukhopadhyay/EDMC-Discord-Presence/releases/tag/v2.1.0) release for backwards compatibility.
